shared/checks/oval/dconf_gnome_banner_enabled.xml

```diff
@@ -10,6 +10,7 @@
     </metadata>
     <criteria operator="OR">
       <extend_definition comment="dconf installed" definition_ref="package_dconf_installed" negate="true" />
+      <extend_definition comment="gdm installed" definition_ref="package_gdm_installed" negate="true" />
```

Comment:
Should return not applicable if the package is not installed; otherwise, return pass/fail if configured configured correctly or not. See #2272 for what we most likely need to do. We should do this for all DConf checks.

Reply:
@redhatrises #2272 does not return `notapplicable` if package is not installed. It checks if package is required, not required, or unset (doesn't matter) through Value `sshd_required`. Which means:
- required: check that package is installed and configured (The policy says you should have it installed and configured)
- not required: check that package is not installed (The policy says you should not have it installed)
- doesn't matter: check state of package; (The policy gives freedom to each machine to have it installed or not)
  - if installed, check configuration
  - if not installed, pass

It is like a guideline from the policy to the rule.

So, for `dconf_gnome_*` Rules, is it enough to determine if Rule should be evaluated or result in not applicable based just on a certain package installed?
IOW, shoud the policy allow freedom to have the package installed or not?
If yes, using `applicability_check` attributes definition should be enough.

Reply:
@yuumasato if package gdm is not installed, should return not applicable; otherwise if package gdm is installed, pass/fail check.

Reply:
From discussion here, https://github.com/OpenSCAP/openscap/issues/273 `applicability_check` may not be the way to go...
Not sure how to proceed.